manuscripts2/derived_classes.py

```diff
@@ -0,0 +1,14 @@
+from new_functions import Query
```

Comment:
You need to add the license/copyright template. Please, get it from some other file, and include it here.

Reply:
Okay, I'll add it to the file.